src/btree/bt_debug.c

```diff
@@ -1726,8 +1726,15 @@ __debug_ref(WT_DBG *ds, WT_REF *ref)
             WT_RET(ds->f(ds, "%s", "internal"));
         if (F_ISSET(ref, WT_REF_FLAG_LEAF))
```

Comment:
Earlier it is possible to have different flag options at a time. Now with these two flags, it is only possible to have either `WT_REF_FLAG_INTERNAL` or `WT_REF_FLAG_LEAF`. Can you change this to an `if ... else`?